src/Couchbase.Lite.Shared/API/Sync/URLEndpoint.cs

```diff
@@ -83,7 +83,9 @@ public URLEndpoint([NotNull]Uri url)
 
         void IEndpointInternal.Visit(ReplicatorConfiguration config)
         {
+            #if !NET6_0_WINDOWS10_0_17763_0
             config.RemoteUrl = Url;
```

Comment:
How is this getting handled in the .NET 6 case?

Reply:
Handled in the ReplicatorConfiguration record constructor.